week-2/mandatory/2-ecommerce-db/task.md

```diff
@@ -7,8 +7,79 @@ In this homework, you are going to work with an ecommerce database. In this data
 Below you will find a set of tasks for you to complete to set up a database for an e-commerce app.
 
 To submit this homework write the correct commands for each question here:
+
 ```sql
 
+1. SELECT name, address FROM customers WHERE country='United States';
+
+2. SELECT * FROM customers ORDER BY name ASC;  -- it already orders asc by default --
+
+3. SELECT * FROM products WHERE product_name LIKE '%socks%';
+
+4. SELECT products.id, products.product_name, pa.unit_price, pa.supp_id
+    FROM products JOIN product_availability pa ON products.id=pa.prod_id
+    WHERE pa.unit_price > 100;
```

Comment:
nice work, best practice is to follow one way in giving aliases for tables, so you give products an alias as well